cmd/sink/event_data.go

```diff
@@ -0,0 +1,138 @@
+// Copyright KubeArchive Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+
+	cloudevents "github.com/cloudevents/sdk-go/v2"
+	"github.com/cloudevents/sdk-go/v2/types"
+)
+
+const (
+	apiVersionExtension = "apiversion"
+	kindExtension       = "kind"
+	nameExtension       = "name"
+	namespaceExtension  = "namespace"
+
+	writeQuery = "INSERT INTO ArchiveMeta (api_version, kind, name, namespace, resource_version, data, created_ts, updated_ts) Values ($1, $2, $3, $4, $5, $6, $7, $8)"
+)
+
+// Represents the fields in a resource's metadata that are need to write a resource into the database
```

Comment:
Tense, should be "needed":

```suggestion
// Represents the fields in a resource's metadata that are needed to write a resource into the database
```

Reply:
fixed